h2o-core/src/test/java/water/TestUtil.java

```diff
@@ -651,4 +658,13 @@ public void done(Frame frame) {
     }
   }
 
+  @BeforeClass
+  public static void hi() {
+    stall_till_cloudsize(1);
+    Scope.enter();
+  }
+
+  @AfterClass public static void bye() { Scope.exit(); }
+
+  protected static Vec willDrop(Vec v) { return Scope.track(v); }
```

Comment:
A lot of sense. Scope is an extra entity; we don't have to remember all these words. More, there are two willDrop methods, one takes a Vec, another takes a reference to Vec. Let's hide this complexity.